tests/js/chartAuthoringTests.js

```diff
@@ -21,10 +29,14 @@ https://github.com/floe/universal/LICENSE.txt
         },
         components: {
             dataEntryPanel: {
+                container: ".floec-dataEntryPanel"
```

Comment:
When comment https://github.com/fluid-project/chartAuthoring/pull/7#discussion_r40561201 is addressed, this line and line 35 can be removed.